knowledge-base/how-to-use-kendodatetimepicker-for-parameter-editors-in-the-angular-report-viewer.md

```diff
@@ -0,0 +1,117 @@
+in [Angular Report Viewer](./angular-report-viewer). This approach allows the selection of date and time for the **DateTime** [report parameters](./designing-reports-parameters).
+
+
+## Solution
+Telerik kendoDateTimePicker widget is not included in the Kendo UI JavaScript distributed by 
+Telerik Reporting through telerikReportViewer.kendo-x.x.x.min.js file, or kendo.subset.2015.3.930.min.js in older versions.
+For that reason, you will have to replace the Kendo UI subset with the full Kendo UI JavaScript e.g. kendo.all.min.js.
+
+Also, you need to make sure that Kendo all is loaded after jQuery. Because of that, we will use a special logic that sets the visibility
+of the viewer after Kendo all is loaded:
+
+1. In the initializaion of the viewer, define the parameterEditors [option](./angular-report-viewer-api-options).
+```TypeScript
+<tr-viewer #viewer1 *ngIf="visible"
+    [containerStyle]="viewerContainerStyle"
+    [serviceUrl]="'http://localhost:59654/api/reports/'"
+    [reportSource]="{
+        report: 'MyReport.trdp',
+        parameters: {}
+    }"
+    [parameterEditors]="[{
+        match: match,
+        createEditor: createEditor
+    }]"
+</tr-viewer>
+```
+
+2.Then in the **app.component.ts** implement the createEditor function:
+
+```TypeScript
+export class AppComponent implements OnInit {
+    @ViewChild('viewer1', { static: false }) viewer: TelerikReportViewerComponent;
+
+    private visible: boolean = false;
+    . . .
+
+    ngOnInit() {
+        this.loadScript("http://kendo.cdn.telerik.com/2020.3.1118/js/kendo.all.min.js")
+            .then(() => {
+                this.visible = true;
+            });
+    }
+
+   . . .
+    match(parameter) {
+        return parameter.type === "System.DateTime";
+    }
+
+    createEditor (placeholder, options){
+        var dateTimePicker = $(placeholder).html('<input type="datetime"/>'),
+            parameter,
+            valueChangedCallback = options.parameterChanged,
+            dropDownList;
+
+        function onChange() {
+            var val = dropDownList.value();
+            valueChangedCallback(parameter, val);
+        }
+
+        return {
+            beginEdit: function (param) {
+
+                parameter = param;
+
+                $(dateTimePicker).find("input").kendoDateTimePicker({
+                    dataTextField: "name",
+                    dataValueField: "value",
+                    value: parameter.value,
+                    dataSource: parameter.availableValues,
+                    change: onChange
+                });
+
+                dropDownList = $(dateTimePicker).find("input").data("kendoDateTimePicker");
+            }
+        };
+    }
+}
+```
+
+## See Also
+- [KendoDateTimePicker](https://docs.telerik.com/kendo-ui/api/javascript/ui/datetimepicker)
+- [Angular Report Viewer](./angular-report-viewer)
```

Comment:
When there is are no empty rows between the bullet points, the text seems to get rendered with smaller font size in the live documentation. Test adding empty lines between the bullets e.g.:
- bullet 1
 
- bullet 2